perllib/FixMyStreet/Cobrand/Thamesmead.pm

```diff
@@ -115,4 +115,60 @@ sub geocoder_munge_results {
 
 }
 
+sub report_new_is_on_thamesmead_area {
```

Comment:
I'm not totally sold on the name of this method, but I can't come up with anything better. I think my issue is the name suggests it's going to return a boolean, but it actually returns a category.

I thought about `report_new_is_on_thamesmead_area_and_return_area_type`, but perhaps that's a bit of a mouthful!

Happy to leave this as-is for now, but just flagging in case you had any immediate thoughts on an alternative name.

Reply:
It should probably just be return_area_type, thinking about it?